tests/robottelo/test_func_locker.py

```diff
@@ -0,0 +1,321 @@
+# coding: utf-8
```

Comment:
@rochacbruno thanks! Does it mean that it's better to use airplane pattern for emacs support or `# coding: utf-8` is supported there too?